### DISCUSSION TITLE
Help with checking my code for logical errors

### DISCUSSION THREAD

**Discussion by incposecretariate** [community/community#78471] [Programming Help]
### Body

Can someone help with doing a go through of my code. Would be happy to learn from it.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by incposecretariate** [comment]
@Iron-Ham @mgriffin can you help me get around the community and my code

**Reply by deep100x** [reply]
Tell me, I wil help you.

**Reply by incposecretariate** [reply]
in my code.gs code I have a problem in execution. That is not posting to my GoogleSheets, which previously was but the file upload was not working. @deepk2891 Are you able to view my repo?

**Reply by deep100x** [reply]
Also, i am not getting what is the issue..

**Reply by incposecretariate** [reply]
Now my google sheet posts but not the upload.

**Reply by incposecretariate** [reply]
Thanks now both are posting, but i need to figure out a way to get the url of the stored file and send it to the sheet. so that it can be retrieved later. Also other inputs are to be appended. now the problem is the appendrow is not getting the append in a fixed column order. it is randomizing.

**Reply by incposecretariate** [reply]
Thanks all, now the issues are resolved. The only thing remaining is grouping answers for 7,9, 11
